packages/fast-element/src/template.ts

```diff
@@ -14,38 +14,54 @@ export interface SyntheticViewTemplate {
     create(): SyntheticView;
 }
 
-export class HTMLTemplate extends Directive
+export class ViewTemplate extends Directive

```

Comment:
Renamed for greater clarity (related to interfaces) and to disambiguate from platform template primitives.